src/pages/Inventory.page.js

```diff
@@ -9,6 +9,23 @@ export class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
 
+    get inventoryItemName() { return this.page.locator('.inventory_item_name '); }
+
+    get inventoryItemPrice() { return this.page.locator('.inventory_item_price'); }
+
+    async selectSortOption(value) {
+        await this.productSorting.selectOption(value);
+    }
+
+    async verifySortingNames() {
```

Comment:
This function gets some information from elements, but not verify it. Could be named for example: getSoretingsNames